### PR TITLE
LMB-607 | The besluit does not have to be added to the mandataris anymore

### PR DESCRIPTION
**Pull request by JonasVanHoof**
## Description

As we updated the mandataris/besluit/artikel model we do not have to set the besluit on the  mandataris anymore. When a decision is found for the mandataris the publication status is set to BEKRACHTIGD.

## How to test

REmoved logic for adding the besluit to the mandataris, this logic will again be tested in LMB-666

## Links to other PR's

- https://github.com/lblod/app-lokaal-mandatenbeheer/pull/194 => app
- https://github.com/lblod/frontend-lokaal-mandatenbeheer/pull/311 => frontend

### CODE REVIEW

File: controllers/mandatees-decisions.ts
Comment (Rahien) [inline controllers/mandatees-decisions.ts:1]: while you're at it, can you fix some of the typos here?

dicision, doe snot, and PUBLICATION_STATUS.BEKRACHTIGT (should be d) are some that i see

Comment (Rahien) [reply controllers/mandatees-decisions.ts:1]: :deer: :sneezing_face:

Comment (Rahien) [reply controllers/mandatees-decisions.ts:1]: I also noticed that findDecisionForMandataris uses the incorrect ext: prefix for bekrachtigtAanstellingVan. It should be mandaat.

**Review by Rahien — CHANGES_REQUESTED** [review]

File: controllers/mandatees-decisions.ts
Comment (JonasVanHoof) [reply controllers/mandatees-decisions.ts:1]: Does ABB provide us with language lessons?

Also added a todo to look for mandaat:bekractigdOntslagVan

**Review by Rahien — APPROVED** [review]

**Pull request merged**